frontend/src/concepts/pipelines/content/import/PipelineImportModal.tsx

```diff
@@ -28,11 +30,14 @@ const PipelineImportModal: React.FC<PipelineImportModalProps> = ({ isOpen, onClo
   const [error, setError] = React.useState<Error | undefined>();
   const [{ name, description, fileContents, pipelineUrl, uploadOption }, setData, resetData] =
     usePipelineImportModalData();
+  const [{ items: pipelines }] = useAllPipelines();
```

Comment:
do we need to fetch all the pipelines for this. can we get away with just doing a regular fetch pipelines call with a filter?

Reply:
I was thinking we weren't able to filter by name and only ID, but you are correct and I have updated this. The recursive fetching is required for the versions displayed in the table though. I've scrapped the others.